test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpImplicitExpressionTest/ParsesNullConditionalOperatorImplicitExpression_Dot11.stree.txt

```diff
@@ -1,5 +1,8 @@
-Expression block - Gen<Expr> - 4 - (0:0,0)
-    Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-        SyntaxKind.Transition;[@];
-    Code span - Gen<Expr> - [val] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (1:0,1) - Tokens:1
-        SyntaxKind.Identifier;[val];
+CSharpCodeBlock - [0..4)::4 - [@val]
```

Comment:
something has to be wrong

Reply:
Gonna stop commenting, but there's a few other test files that seem too simple

Reply:
This is because these tests call `ImplicitExpressionTest` which calls `ParseBlockTest` as opposed to `ParseDocumentTest` https://github.com/aspnet/Razor/blob/master/test/Microsoft.AspNetCore.Razor.Language.Test/Legacy/CodeParserTestBase.cs#L63. So the parsing basically stops after it parses an implicit expression. This was the same behavior we had in the old world.

Reply:
I hate our tests 😢